### DISCUSSION TITLE
framer-motion version upgrade issue

### DISCUSSION THREAD

**Discussion by ganeshkrishnasharma** [chakra-ui/chakra-ui#5531] [Q&A]
Hi All, If i try to upgrade framer-motion to any higher version above 4.1.17, I end up getting the following error when starting the react project. 

Failed to compile.

./node_modules/framer-motion/dist/es/components/AnimatePresence/index.mjs
![Screenshot 2022-02-05 at 9 52 30 AM](https://user-images.githubusercontent.com/39894982/152628510-ac0acacc-e9b9-4380-8e9f-bfc5c5fcd1db.png)

Can't import the named export 'Children' from non EcmaScript module (only default export is available)

Anyone encountered same issue or has been able to upgrade framer-motion to latest version?

**Comment by annasmbr** [comment]
Downgrade Framer motion version to “4.1.17” in your package.json file:
 "framer-motion": "4.1.17",
run npm install
